doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -182,8 +182,31 @@ To bring a stalled Target node to idle state, use the ``mesh models dfu cli canc
 
 .. _bluetooth_mesh_dfu_eval_md:
 
-Composing the firmware metadata
-*******************************
+Generating the firmware metadata
+********************************
+
+Automated metadata generation
+=============================
+
+The most efficient way to generate the required DFU metadata is to use the DFU metadata extraction script integrated in the NCS build system. By enabling the ``CONFIG_BT_MESH_DFU_METADATA_ON_BUILD`` option in the
+application, the metadata will be automatically parsed from the .elf and .config file. The parsed data is put into a DFU zip-archive in the build folder of the application in a JSON format. Additionally, the
```

Comment:
We can say the archive and file name. Other FOTA pages uses the archive name as well.
```suggestion
application, the metadata will be automatically parsed from the .elf and .config file. The parsed data is stored in `ble_mesh_metadata.json` and put into a `dfu_application.zip` archive in the build folder of the application. Additionally, the
```

Reply:
@Balaklaka While you are correct, I think it just becomes more confusing if we add `dfu_application.zip_ble_mesh_metadata.json`. I think it is easier if the user just have to focus on what is inside the zip.